ansible/roles_ocp_workloads/ocp4_workload_dso/tasks/infrastructure.yml

```diff
@@ -25,7 +25,7 @@
   become: true
 
 - name: retrieve registry route
-  shell: "{{ ocp4_dso_openshift_cli }} get routes --all-namespaces | grep registry | grep default | grep -v console | awk '{print $3}'"
+  shell: "{{ ocp4_dso_openshift_cli }} get routes --all-namespaces | grep image-registry | awk '{print $3}'"
```

Comment:
Please move this to the k8s_info module. We do not want to keep using `shell` and `command`.

Something like:

```
- k8s_info:
      api_version: imageregistry.operator.openshift.io/v1
      kind: Config
      name: cluster
    register: something
```

Reply:
@stencell Thank you, updated